doc/src/Build_manual.rst

```diff
@@ -76,66 +98,79 @@ available:
 
 ----------
 
-Installing prerequisites for HTML build
-=======================================
+Build using CMake
+-----------------
+
+It is also possible to create the HTML version (and **only** the HTML
+version) of the manual within the :doc:`CMake build directory
+<Build_cmake>`.  The reason for this option is to include the
+installation of the HTML manual pages into the "install" step when
+installing LAMMPS after the CMake build via ``cmake --build . --target
+install``.  The documentation build is included in the default build
+target, but can also be requested independently with
+``cmake --build . --target doc``.
+
+.. code-block:: bash
+
+   -D BUILD_DOC=value       # yes or no (default)
+
+----------
+
+Prerequisites for HTML
+----------------------
 
 To run the HTML documentation build toolchain, python 3, git, doxygen,
 and virtualenv have to be installed locally.  Here are instructions for
 common setups:
 
-Ubuntu
-------
+.. tabs::
 
-.. code-block:: bash
+   .. tab:: Ubuntu
 
-   sudo apt-get install python-virtualenv git doxygen
+      .. code-block:: bash
 
-Fedora (up to version 21) and Red Hat Enterprise Linux or CentOS (up to version 7.x)
-------------------------------------------------------------------------------------
+         sudo apt-get install python-virtualenv git doxygen
 
-.. code-block:: bash
+   .. tab:: RHEL or CentOS (Version 7.x)
 
-   sudo yum install python3-virtualenv git doxygen
+      .. code-block:: bash
 
-Fedora (since version 22)
--------------------------
+         sudo yum install python3-virtualenv git doxygen
 
-.. code-block:: bash
+   .. tab:: Fedora or RHEL/CentOS (8.x or later)
 
-   sudo dnf install python3-virtualenv git doxygen
+      .. code-block:: bash
 
-MacOS X
--------
+         sudo dnf install python3-virtualenv git doxygen
 
-Python 3
-^^^^^^^^
+   .. tab:: MacOS X
 
-Download the latest Python 3 MacOS X package from
-`https://www.python.org <https://www.python.org>`_
-and install it.  This will install both Python 3
-and pip3.
+      *Python 3*
 
-virtualenv
-^^^^^^^^^^
+      Download the latest Python 3 MacOS X package from
+      `https://www.python.org <https://www.python.org>`_ and install it.
+      This will install both Python 3 and pip3.
 
-Once Python 3 is installed, open a Terminal and type
+      *virtualenv*
 
-.. code-block:: bash
+      Once Python 3 is installed, open a Terminal and type
+
+      .. code-block:: bash
 
-   pip3 install virtualenv
+         pip3 install virtualenv
```

Comment:
We should think about updating this to the `venv` module that is part of Python 3. That way we avoid having to install `virtualenv`.

Reply:
won't be able to do this in this PR. too many ripples.